mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports: True
```

Comment:
How many of these do we have? I'd prefer being deliberate case by case knowing which external modules cannot be type checked instead of skipping silently. It would give us a better confidence about our type safety.

Reply:
343 vs 272. Not sure how many actual modules though. I can try to trawl through the output to pick them out individually

Reply:
I think I agree with @fpacifici here — I think blacklisting specific modules in the mypy configuration would be a better bet:

> To suppress all missing import imports errors from a single library, add a section to your mypy config file for that library setting `ignore_missing_imports` to True. For example, suppose your codebase makes heavy use of an (untyped) library named `foobar`. You can silence all import errors associated with that library and that library alone by adding the following section to your config file:
> ```
> [mypy-foobar]
> ignore_missing_imports = True
> ```
> Note: this option is equivalent to adding a `# type: ignore` to every import of foobar in your codebase. For more information, see the documentation about configuring import discovery in config files.

(from https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-type-hints-for-third-party-library)

Reply:
I've updated this